web/src/message_list_tooltips.ts

```diff
@@ -124,21 +123,9 @@ export function destroy_all_message_list_tooltips(): void {
     message_list_tippy_instances.clear();
 }
 
-function get_last_edit_timestr(message: Message): string | null {
-    let last_edit_timestamp;
-    if (message.local_edit_timestamp !== undefined) {
-        last_edit_timestamp = message.local_edit_timestamp;
-    } else {
-        last_edit_timestamp = message.last_edit_timestamp!;
-    }
```

Comment:
The `local_edit_timestamp` code here was related to locally echoed edits; I'm not sure we can remove that safely.

Reply:
We have the exact same if/else block when we set `message_container.last_edit_timestamp` in `message_list_view`. Since we have the `message_container` in the `initialize` function that calls this below, it seemed better to use that instead of checking the message object again for those two fields.

Reply:
Tested with throttling the browser in the dev environment to make sure we still show the tooltip for the "SAVING" indicator that we show when there is a local edit timestamp.